robot/EDA/resources/ContactsPageObject.py

```diff
@@ -171,7 +171,18 @@ def open_item(self, locator, error_message, capture_screen):
         self.selenium.click_element(locator)
         if capture_screen:
             self.selenium.capture_page_screenshot()
-            
+
+    def open_apex(self, title, error_message, capture_screen):
+        """ Performs a wait until the element shows on the page, and clicks the element """
+        self.selenium.wait_until_page_contains_element(
+            contacts_locators["apex_jobs"].format(title), 
+            timeout=60,
+            error=error_message
+        )
+        self.selenium.click_element(contacts_locators["apex_jobs"].format(title))
+        if capture_screen:
+            self.selenium.capture_page_screenshot()
```

Comment:
I suppose robot already takes screenshots for failed tests right? Does this method capture page screenshot irrespective of the test status(pass/fail)?